src/dotnet/AzureAI.Proxy/ReverseProxy/ThrottlingHealthPolicy.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Yarp.ReverseProxy.Health;
```

Comment:
For my understanding, what is the flow if you get a 429 on the first endpoint? Will it retry once again on another healthy endpoint and mark the endpoint as unhealthy until the retry-after is done?

Or will it keep retrying on other endpoints?

Reply:
it will keep retrying until there is nothing found anymore - priority can be any value, not just 1 or 2